impeller/entity/contents/content_context.h

```diff
@@ -774,7 +774,10 @@ class ContentContext {
     // The prototype must always be initialized in the constructor.
     FML_CHECK(prototype != container.end());
 
-    auto variant_future = prototype->second->WaitAndGet()->CreateVariant(
+    auto pipeline = prototype->second->WaitAndGet();
+    FML_CHECK(pipeline);
```

Comment:
We should return nullptr here like we do above. I think its possible to get null if we run into device loss.

Reply:
done